### DISCUSSION TITLE
Gauge Status

### DISCUSSION THREAD

**Discussion by jbadeau** [getgauge/gauge#1923] [General]
Hello,

As ThoughtWorks is no longer supporting Gauge. Has anyone stepped in to keep development active or is it time to migrate to cucumber?

**Comment by zabil** [comment]
Like other open source projects Gauge is maintained by core contributors outside our core working hours. So yes it is active.

> is it time to migrate to cucumber

I am not sure I can comment on this as Cucumber and Gauge are very different tools.

**Reply by jbadeau** [reply]
I was joking about cucumber :p. Just wanted to know if there is anyone maintaining gauge

**Reply by zabil** [reply]
> if there is anyone maintaining gauge

Short answer is yes. 

However work is limited to critical bugs, library updates and security fixes.

**Comment by wlad** [comment]
@jbadeau are you aware of [Robot Framework](https://robotframework.org/)? Your question might be as well " is it time to migrate to Robot?" :wink:

**Comment by Snooz82** [comment]
@zabil Was there an official announcement of ThoughtWorks that i missed?

**Reply by zabil** [reply]
are you referring to something like this? https://github.com/getgauge/gauge/issues/1732
